WordPress/Classes/Utility/Analytics/WPAppAnalytics.m

```diff
@@ -171,7 +171,7 @@ - (void)accountSettingsDidChange:(NSNotification*)notification
 
 + (NSInteger)sessionCount
 {
-    return [[NSUserDefaults standardUserDefaults] integerForKey:WPAppAnalyticsKeySessionCount];
+    return [[UserPersistentStoreFactory userDefaultsInstance] integerForKey:WPAppAnalyticsKeySessionCount];
```

Comment:
Should we leave this file untouched? These defaults seem to be session related. Though when we copy the defaults  once, we wipe the former `standard` data. These would be lost in that case as well. A bit tricky to decide.

Reply:
Just taking a quick look around, I see that there is a specific metric that only fires once based off the session count. I'm not sure if there's others but since there's at least one, maybe we should leave it as is?